src/main/java/org/broadinstitute/hellbender/tools/spark/sv/KmerHistogrammer.java

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.tools.spark.utils.MapPartitioner;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import scala.Tuple2;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Tool to describe reads that support a hypothesis of a genomic breakpoint.
```

Comment:
Can you expand on what this tool is and what it's supposed to do? Looks like it's mainly for debugging?